terraform/groups/environments/policy.tf

```
@@ -0,0 +1,5 @@
+resource "aws_iam_user_policy" "user_policy" {
+  name = "${var.service}-${var.environment}"
+  policy = data.aws_iam_policy_document.user.json
+  user = aws_iam_user.user.name
+}
```

As above, this block would read better if passed through `terraform fmt`:

```suggestion
resource "aws_iam_user_policy" "user_policy" {
  name   = "${var.service}-${var.environment}"
  policy = data.aws_iam_policy_document.user.json
  user   = aws_iam_user.user.name
}
```